src/app/search/SearchBar.tsx

```diff
@@ -459,102 +513,107 @@ function SearchBar(
   );
 
   return (
-    <div
-      className={clsx(className, 'search-filter', styles.searchBar, { [styles.open]: isOpen })}
-      role="search"
-    >
-      <AppIcon icon={searchIcon} className="search-bar-icon" {...getLabelProps()} />
-      <input
-        {...getInputProps({
-          onBlur,
-          onFocus,
-          onKeyDown,
-          ref: inputElement,
-          className: clsx('filter-input', { [styles.invalid]: !valid }),
-          autoComplete: 'off',
-          autoCorrect: 'off',
-          autoCapitalize: 'off',
-          spellCheck: false,
-          autoFocus,
-          placeholder,
-          type: 'text',
-          name: 'filter',
-        })}
-        enterKeyHint="search"
-      />
-      <LayoutGroup>
-        <AnimatePresence>
-          {children}
-
-          {liveQuery.length > 0 && saveable && (
-            <motion.button
-              layout
-              exit={{ scale: 0 }}
-              initial={{ scale: 0 }}
-              animate={{ scale: 1 }}
-              key="save"
-              type="button"
-              className={clsx(styles.filterBarButton, styles.saveSearchButton)}
-              onClick={toggleSaved}
-              title={t('Header.SaveSearch')}
-            >
-              <AppIcon icon={saved ? starIcon : starOutlineIcon} />
-            </motion.button>
-          )}
+    <>
+      <div
+        className={clsx(className, 'search-filter', styles.searchBar, { [styles.open]: isOpen })}
```

Comment:
All this is just a new fragment for the armory portal, the rest is whitespace change.